installers/npm/package.json

```
@@ -29,11 +29,11 @@
   },
   "engines": {
     "node": ">=14 <15",
-    "npm": "7.x"
+    "npm": "8.x"
```

I don't think we want this.  In fact, we might even want `>= 6`?

```suggestion
    "npm": ">= 7.x"
```